website/api/helpers/merge-json-tables-schema.js

```diff
@@ -0,0 +1,133 @@
+module.exports = {
+
+
+  friendlyName: 'Merge JSON Schema Tables',
+
+
+  description: 'Merges the osquery schema JSON with Fleet\'s overrides, and returns the merged JSON array',
```

Comment:
Imperative mood

i.e. 
> "**Merge** the…overrides, and **return**"

Reply:
Probably worth pointing out where it's getting the files from.  For example, something like this:

> Get the extended osquery schema and documentation supported by Fleet by reading the raw osquery tables and Fleet's overrides from disk, then returning the extended set of tables.